packages/core/stories/toolkit-provider.doc.stories.mdx

```diff
@@ -96,6 +96,18 @@ It requires no configuration, but is included in your app when you use a Toolkit
 AriaAnnouncer is ever required, only the outermost instance of ToolkitProvider will instantiate the AriaAnnouncer. If you use
 nested ToolkitProviders, these will not inject further instances of AriaAnnouncer.
 
+## useDensity
+
+You can use `useDensity` hook to access the currently active density of your app in the provided scope.
+
```

Comment:
Do we want to add code examples of these hooks or the API of them?

Reply:
Will do 👍